docker-compose.yml

```
@@ -0,0 +1,33 @@
+services:
+
+  sphinx:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: sphinx
+    volumes:
+      - ./source:/docs/source
+      - docs_build:/docs/_build
+    environment:
+      - ENVIRONMENT=${ENVIRONMENT}
+      - LOCAL_PORT=${LOCAL_PORT}
+      - PRODUCTION_HOST=${PRODUCTION_HOST}
+      - STAGING_HOST=${STAGING_HOST}
+      - DEVELOPMENT_HOST=${DEVELOPMENT_HOST}
+    command: sphinx-autobuild -b dirhtml /docs/source /docs/_build
+    
+  nginx:
+    image: nginx:1.27.3-alpine3.20
```

I guess this could be less restrictive since you use python `python:3.13` and not `3.13.1`.
```suggestion
    image: nginx:1.27-alpine3.20
```